### PR TITLE
Bulgarian Translation Update 5

### PR DESCRIPTION
**Pull request by Effektus**

### CODE REVIEW

File: files/lang/bg.po [hunk 1]
```diff
@@ -19,7 +19,7 @@ msgstr ""
 "X-Generator: Poedit 3.0.1\n"
 
 msgid "MIN"
-msgstr ""
+msgstr "МИНИМАЛНО"
```
Comment (zenseii) [inline files/lang/bg.po:22]: This needs to be 3 letters max to fit on the "MIN" button:
image

File: files/lang/bg.po [hunk 2]
```diff
@@ -2236,17 +2236,19 @@ msgid ""
...
+"замъка, когато няма герой."
 
 msgid ""
 "The Mage Guild allows heroes to learn spells and replenish their spell "
 "points."
 msgstr ""
 
 msgid "Recruit %{name}"
-msgstr ""
+msgstr "Вербувайте"
```
Comment (zenseii) [inline files/lang/bg.po:2248]: Is there a specific reason the %{name} is missing here?

File: files/lang/bg.po [hunk 3]
```diff
@@ -2442,28 +2444,28 @@ msgid "Are you sure you want to dismiss this army?"
...
-msgstr ""
+msgstr "Защита"
 
 msgid "Shots"
-msgstr ""
+msgstr "Истрели"
 
 msgid "Shots Left"
-msgstr ""
+msgstr "Оставащи Истрели"
```
Comment (zenseii) [inline files/lang/bg.po:2456]: It seems like the word Истрели starts with a capital letter? Is this consistent with the other similar terms like Hit points etc?

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi @Effektus ! Thank you again for your continued work on this translation. I left a few comments and I hope you can find the time to have a look at them should you want to.

File: files/lang/bg.po [hunk 4]
```diff
@@ -2877,19 +2891,19 @@ msgid "Move Points:"
...
 msgid "Number to buy:"
-msgstr ""
+msgstr "Покупка на:"
 
 msgid "Click to apply the selected resolution."
 msgstr ""
 
 msgid "MAX"
-msgstr ""
+msgstr "МАК"
```
Comment (zenseii) [inline files/lang/bg.po:2906]: This MAX appears in a different location than MIN, so making it 3 letters isn't strictly necessary, but if what you wrote works then don't bother changing it:
image

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi @Effektus,
I've checked the latest additions.
While testing some of them ingame I noticed this as well:
image

File: files/lang/bg.po [hunk 5]
```diff
@@ -448,10 +447,10 @@ msgid "Cast %{spell}"
 msgstr "Направи %{spell}"
 
 msgid "Select spell target"
-msgstr "Изберете цел за заклинание"
+msgstr "Избери цел за заклинание"
 
 msgid "View Ballista info"
-msgstr "Вижте информация за Катапулт"
+msgstr "Виж информация за Катапулт"
```
Comment (zenseii) [inline files/lang/bg.po:453]: Note that this refers to the Towers defending your castles by shooting at the enemies.

**Review by zenseii — COMMENTED** [review]
@Effektus, I left one comment here which you can take a look at should you have the time.

File: files/lang/bg.po (hunk 5 above)
Comment (Effektus) [reply files/lang/bg.po:453]: It is reasonable

**Review by zenseii — APPROVED** [review]

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Effektus , thank you so much for your dedication regarding this translation!

**Pull request merged**